designsafe/settings/common_settings.py

```diff
@@ -626,6 +626,10 @@ def guid_filter(record):
 DATACITE_PASS = os.environ.get('DATACITE_PASS')
 DATACITE_SHOULDER = os.environ.get('DATACITE_SHOULDER')
 
+# Clarivate/World of Citation
+WOS_URL = os.environ.get('WOS_URL', 'https://api.clarivate.com/apis/wos-starter/v1/documents')
+WOS_PASS = os.environ.get('WOS_APIKEY')
```

Comment:
Can we default this password to an empty string? That way if it ends up missing for whatever reason, it just takes down the Clarivate endpoint and not the entire site.